src/ripple/core/Stoppable.h

```diff
@@ -411,17 +403,6 @@ class RootStoppable : public Stoppable
     alertable_sleep_until(std::chrono::system_clock::time_point const& t);
 
 private:
-    /*  Notify a root stoppable and children to stop, without waiting.
-        Has no effect if the stoppable was already notified.
-
-        Returns true on the first call to this method, false otherwise.
-
-        Thread safety:
-            Safe to call from any thread at any time.
-    */
-    bool
-    stopAsync(beast::Journal j);
-
     std::atomic<bool> m_prepared{false};
```

Comment:
Random comment: _once_ we migrate to C++20, these should be changed to `[std::atomic_flag](https://en.cppreference.com/w/cpp/atomic/atomic_flag)`. Can you add a `TODO` comment tagged with C++20 to that effect please?